### PR TITLE
Updating MAD_PK_KeyPlots and adding code for testing survminer bug in Test folder

### PR DESCRIPTION
**Pull request by iamstein**

### CODE REVIEW

File: Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd [hunk 1]
```diff
@@ -116,62 +92,58 @@ nca = read.csv("../Data/PPtmp_NCA.csv",stringsAsFactors = FALSE)
...
-                          )},alpha=.5) +  
-  stat_summary(geom = "point", size = 2, fun.y = mean,alpha=.5) + 
-  stat_summary(geom = "line", size = 1, fun.y = mean,alpha=.5) 
-gg = gg + scale.y.log10()  + annotation_logticks(base = 10, sides = "l", color = rgb(0.5,0.5,0.5))
-gg = gg + scale_x_units(units.input = "h",units.output="m",t.end = 6,increment=1) 
-gg = gg + ylab("Conc (ug/ml)")
-gg = gg + labs(color="Dose (mg)")
-gg = gg + facet_grid(~REG)
+gg <- ggplot(data = data, aes(x = NOMTIME, y = LIDV, color = DOSE_high2low, group=DOSE_high2low))
+gg <- gg + xgx_geom_ci()
```
[the remark this answers is not in the document]
Comment (iamstein) [reply Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd]: Done

File: Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd [hunk 2]
```diff
@@ -116,62 +92,58 @@ nca = read.csv("../Data/PPtmp_NCA.csv",stringsAsFactors = FALSE)
...
+gg <- gg + facet_grid(~REGIMEN)
+gg <- gg + xgx_stat_ci(conf_level = .95)
```
[the remark this answers is not in the document]
Comment (iamstein) [reply Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd]: removed one of them.

File: Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd (hunk 1 above)
[the remark this answers is not in the document]
Comment (iamstein) [reply Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd]: added

File: Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd [hunk 3]
```diff
@@ -181,77 +153,83 @@ print(gg)
...
-  stat_summary(geom = "point", size = 2, fun.y = mean,alpha=.5) + 
-  stat_summary(geom = "line", size = 1, fun.y = mean,alpha=.5) 
-gg = gg + scale.y.log10()  + annotation_logticks(base = 10, sides = "l", color = rgb(0.5,0.5,0.5))
-gg = gg + scale_x_units(units.input = "h",units.output="m",t.end = 6,increment=1) 
-gg = gg + labs(color="Dose (mg)")
-gg = gg + facet_grid(~REG)
-gg = gg + ylab("Normalized Conc (ug/ml/mg)")
-gg = gg + ggtitle("Normalized Concentration")
+gg <- ggplot(data = data, aes(x = NOMTIME, y = LIDVNORM, color = DOSE_high2low, group=DOSE_high2low))
+gg <- gg + xgx_stat_ci(conf_level = .95)
```
[the remark this answers is not in the document]
Comment (iamstein) [reply Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd]: done

File: Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd [hunk 4]
```diff
@@ -116,62 +92,58 @@ nca = read.csv("../Data/PPtmp_NCA.csv",stringsAsFactors = FALSE)
...
-gg = gg + labs(color="Dose (mg)")
-gg = gg + facet_grid(~REG)
+gg <- ggplot(data = data, aes(x = NOMTIME, y = LIDV, color = DOSE_high2low, group=DOSE_high2low))
+gg <- gg + xgx_geom_ci()
+gg <- gg + facet_grid(~REGIMEN)
+gg <- gg + xgx_stat_ci(conf_level = .95)
+gg <- gg + xgx_scale_y_log10() 
+gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 
+                                  units_plot    = time_units_plot)
+gg <- gg + labs(y=conc_label,color = trtact_label)
```
[the remark this answers is not in the document]
Comment (iamstein) [reply Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd]: I will try...

File: Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd [hunk 5]
```diff
@@ -18,95 +18,71 @@ Data specifications can be accessed on [Datasets](Datasets.html) and Rmarkdown t
...
-#ggplot settings
-  theme_set(theme_bw(base_size=12))
+# ggplot settings
+  xgx_theme_set()
   
-#xgx functions
-  source("../R/xgx_packages_functions.R")
+# directories for saving individual graphs
+dirs = list(
+  parent_dir= tempdir(),
```
[the remark this answers is not in the document]
Comment (iamstein) [reply Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd]: done.  i changed it in Singe_Ascending_Dose too.

File: Rmarkdown/Multiple_Ascending_Dose_PKPD_time_to_event.Rmd [hunk 6]
```diff
@@ -69,7 +69,7 @@ Time-to-event plots can be summarized by Kaplan-Meier plots and stratified by ex
 Yang, Jun, et al. "The combination of exposure‐response and case‐control analyses in regulatory decision making." The Journal of Clinical Pharmacology 53.2 (2013): 160-166.
 
 ```{r warning=FALSE, message=FALSE}
-km_fit <- survfit(Surv(time) ~ exposure, data = km_data, conf.int = 0.95)
+km_fit <- survfit(Surv(time, status) ~ exposure, data = km_data, conf.int = 0.95)
```
[the remark this answers is not in the document]
Comment (iamstein) [reply Rmarkdown/Multiple_Ascending_Dose_PKPD_time_to_event.Rmd]: Done

File: Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd [hunk 7]
```diff
@@ -8,14 +8,13 @@ output:
 ---
 
 ### Overview
-This document contains PK exploratory graphs and also the R code that generates these graphs, starting from the `dzz` and `PPtmp` files that are stored here:
+This document contains PK exploratory graphs and also the R code that generates these graphs, starting from the `dzz` and `PPtmp` files that are stored at the links below.  For detailed thougths on how to interpret these graphs, please consult the other xGx PK pages for [Single Ascending Dose](Single_Ascending_Dose_PK.html) and [Multiple Ascending Dose](Multiple_Ascending_Dose_PK.html)
```
Comment (margoal1) [inline Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd]: "thougths" should be "thoughts"

File: Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd [hunk 8]
```diff
@@ -123,15 +122,15 @@ nca = nca %>%
...
-gg <- gg + xgx_stat_ci(conf_level = .95)
 gg <- gg + xgx_scale_y_log10() 
 gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 
                                   units_plot    = time_units_plot)
-gg <- gg + labs(y=conc_label,color = trtact_label)
+gg <- gg + labs(y=conc_label, color = trtact_label)
 gg <- gg + xgx_annotate_status(status)
 gg <- gg + xgx_annotate_filenames(dirs)
+gg1<- gg
+gg <- gg + xgx_stat_ci(conf_level = 0.95, geom = list("point","line","errorbar"), alpha = 0.5, position = position_dodge(width = 24*5))
```
Comment (margoal1) [inline Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd]: Should we break up long lines over multiple lines?

File: Rmarkdown/Single_Ascending_Dose_PK.Rmd [hunk 9]
```diff
@@ -27,7 +27,7 @@ xgx_theme_set()
 
 #directories for saving individual graphs
 dirs = list(
-  parent_dir= tempdir(),
+  parent_dir = "Parent_Directory",
```
Comment (margoal1) [inline Rmarkdown/Single_Ascending_Dose_PK.Rmd]: OK, but figures in www folder still have old directory (need to clear cache to update)

File: Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd [hunk 10]
```diff
@@ -24,7 +23,7 @@ Data specifications can be accessed on [Datasets](Datasets.html) and Rmarkdown t
   library(tidyr)
   library(dplyr)
   library(stringr)
-  library(xgxr, lib.loc = "../Rlib")
+  library(xgxr)#, lib.loc = "../Rlib")
```
Comment (margoal1) [inline Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd]: Why is there "#"? Was this to use the v1.0.7? Should we just install v1.0.7 in Rlib now?

**Pull request merged**